packages/workflow/src/records/webhooks.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import fetch from 'node-fetch'
+import { EVENT_TYPE, RegisteredRecord } from '@opencrvs/commons/types'
+import { WEBHOOKS_URL } from '@workflow/constants'
+
+const WEBHOOK_URLS = {
+  [EVENT_TYPE.BIRTH]: new URL('/events/birth/mark-registered', WEBHOOKS_URL),
+  [EVENT_TYPE.DEATH]: new URL('/events/death/mark-registered', WEBHOOKS_URL),
+  [EVENT_TYPE.MARRIAGE]: new URL(
+    '/events/marriage/mark-registered',
+    WEBHOOKS_URL
+  )
```

Comment:
If I remember correctly, we don't have webhook endpoints for Marriage event yet. So this could cause all marriage registrations to fail

Reply:
that's a very good point,  looking into it, there actually is and it just always returns 200: https://github.com/opencrvs/opencrvs-core/blob/cbe8994c19c67550e92120a0d381713f7e2d84c4/packages/webhooks/src/features/event/handler.ts#L224